### PR TITLE
WRN-1542: Sampler: resolve the warning related to babel

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
Resolve the warning related to babel

### Resolution
As logs described, added "loose" option in `@babel/preset-env``.

> Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-methods since the "loose" mode option was set to "true" for @babel/plugin-proposal-class-properties. The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding ["@babel/plugin-proposal-private-methods", { "loose": true }] to the "plugins" section of your Babel config.

### Additional Considerations
We removed duplicated plugins.
The "storybook-utils/configs/Babel.js" file has additional plugins to compile files.
But as the research, some plugins are already included in @babel/preset-env v7.15.0.
- @babel/plugin-proposal-export-namespace-from:	`@babel/preset-env@7.11.0(31 Jul 2020)`
- @babel/plugin-proposal-numeric-separator:	`@babel/preset-env@v7.9.0(21 Mar 2020)`
- @babel/plugin-syntax-dynamic-import:	`@babel/preset-env@v7.5.0(6 Jul 2019)`
- @babel/plugin-proposal-class-properties:	`@babel/preset-env@v7.10.0(27 May 2020)`
- @babel/plugin-proposal-optional-chaining:	`@babel/preset-env@v7.8.0(12 Jan 2020)`
- @babel/plugin-proposal-nullish-coalescing-operator:	`@babel/preset-env@v7.8.0(12 Jan 2020)`

### Links
WRN-1542

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## unreleased
+
+* Adds `private-methods`, and `private-property-in-object` Babel plugins with {loose: true} option.
```
Comment (MikyungKim) [inline CHANGELOG.md:3]: Please write the log in past tense.

```suggestion
* Added `private-methods` and `private-property-in-object` Babel plugins to remove warning
```

**Review by MikyungKim — COMMENTED** [review]
I'm still seeing the below warning.
```
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
        ["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
to the "plugins" section of your Babel config.
```

File: configs/babel.js [hunk 2]
```diff
@@ -76,18 +77,13 @@ module.exports = function (api) {
...
 
 			// Stage 3
 			require('@babel/plugin-syntax-dynamic-import').default,
 			// '@babel/plugin-syntax-import-meta',
-			[require('@babel/plugin-proposal-class-properties').default, {loose: true}],
 			// '@babel/plugin-proposal-json-strings'
 
-			// Soon to be included within pre-env; include here until then
-			require('@babel/plugin-proposal-optional-chaining').default,
 			require('@babel/plugin-proposal-nullish-coalescing-operator').default,
```
Comment (MikyungKim) [inline configs/babel.js:87]: we can remove this also.

File: package.json [hunk 3]
```diff
@@ -27,13 +27,9 @@
   },
   "dependencies": {
     "@babel/core": "^7.12.10",
-    "@babel/plugin-proposal-class-properties": "^7.12.1",
     "@babel/plugin-proposal-decorators": "^7.12.1",
     "@babel/plugin-proposal-export-default-from": "^7.12.1",
-    "@babel/plugin-proposal-export-namespace-from": "^7.12.1",
     "@babel/plugin-proposal-nullish-coalescing-operator": "^7.12.1",
```
Comment (MikyungKim) [inline package.json:32]: please remove this.

File: configs/babel.js [hunk 4]
```diff
@@ -76,18 +77,13 @@ module.exports = function (api) {
 			// Stage 2
 			[require('@babel/plugin-proposal-decorators').default, false],
 			// '@babel/plugin-proposal-function-sent',
-			require('@babel/plugin-proposal-export-namespace-from').default,
-			require('@babel/plugin-proposal-numeric-separator').default,
 			// '@babel/plugin-proposal-throw-expressions',
 
 			// Stage 3
 			require('@babel/plugin-syntax-dynamic-import').default,
```
Comment (MikyungKim) [inline configs/babel.js:83]: I found this one too from v.7.12.0.

File: package.json [hunk 5]
```diff
@@ -27,13 +27,9 @@
...
-    "@babel/plugin-proposal-numeric-separator": "^7.12.7",
-    "@babel/plugin-proposal-optional-chaining": "^7.12.7",
     "@babel/plugin-syntax-dynamic-import": "7.8.3",
```
Comment (MikyungKim) [inline package.json:33]: Please remove this too.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,7 @@
+## unreleased
+
+* Remove Babel plugins because these were already included in `babel-preset-env` v7.15.0.
```
Comment (MikyungKim) [inline CHANGELOG.md:3]:
```suggestion
* Removed some Babel plugins which were already included in the recent `@babe/preset-env`.
```

File: configs/babel.js (hunk 2 above)
Comment (hong6316) [reply configs/babel.js]: OK

File: package.json (hunk 3 above)
Comment (hong6316) [reply package.json]: OK

File: configs/babel.js (hunk 4 above)
Comment (hong6316) [reply configs/babel.js]: OK

File: package.json (hunk 5 above)
Comment (hong6316) [reply package.json]: OK

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**